frontend/src/utilities/useNotification.ts

```diff
@@ -37,8 +38,16 @@ const useNotification = (): {
     },
     [dispatch],
   );
+  const [notificationFunc, setNotificationFunc] = React.useState<NotificationFunc>({
+    success,
+    error,
+  });
+
+  React.useEffect(() => {
+    setNotificationFunc({ success, error });
+  }, [success, error]);
```

Comment:
Doh, I know I said this -- `useMemo` here instead. A `useEffect` with only a `setUseStateFunction` is just `useMemo` with more steps. Sorry about that.

eg
```
const notification = React.useMemo(() => ({ success, error }), [success, error])
```

Reply:
Updated.